### DISCUSSION TITLE
monitoring-bulk-validation-tool

### DISCUSSION THREAD

**Discussion by IvanKvesic** [INSPIRE-MIF/helpdesk-validator#1131] [Q&A]
Is this still active bulk processing tool for the inspire reference validator or does it work with the newer version inspire-validator:2024.2 because i can't get results it only creates md.json file?

**Comment by veronicamonaco** [comment]
Dear @IvanKvesic,
could you please tell me which is the value of the variables "**validator_url_0**" and "**validator_url_1**" in the config.properties file?

**Comment by IvanKvesic** [comment]
[config.zip](https://github.com/user-attachments/files/17543587/config.zip)
I am providing you with my entire config.properties file
